CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
```

Comment:
Please add the release header with its markdown link

e.g.

```
## [1.4.2] - 2024-01-09

...
```

and add the link to the bottom of this file

```
[Unreleased]: https://github.com/openfga/openfga/compare/v1.4.2...HEAD
[1.4.2]: https://github.com/openfga/openfga/releases/tag/v1.4.2
```